clients/stellarcore/client.go

```diff
@@ -250,6 +250,11 @@ func (c *Client) GetLedgerEntryRaw(ctx context.Context, ledgerSeq uint32, keys .
 	return resp, c.makeLedgerKeyRequest(ctx, &resp, "getledgerentryraw", ledgerSeq, keys...)
 }
 
+func (c *Client) GetLedgerEntries(ctx context.Context, ledgerSeq uint32, keys ...xdr.LedgerKey) (*proto.GetLedgerEntryResponse, error) {
+	var resp *proto.GetLedgerEntryResponse
```

Comment:
```suggestion
	var resp proto.GetLedgerEntryResponse
```
I think this should not be a pointer